### PR TITLE
[DNM] record the region report time

### PR DESCRIPTION
**Pull request by nolouch**
### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has configuration change
- Has HTTP API interfaces change (Don't forget to [add the declarative for API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tidb-ansible`](https://github.com/pingcap/tidb-ansible):
- Need to cherry-pick to the release branch

### CODE REVIEW

File: server/metrics.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ var (
 			Subsystem: "scheduler",
 			Name:      "handle_region_heartbeat_duration_seconds",
 			Help:      "Bucketed histogram of processing time (s) of handled region heartbeat requests.",
-			Buckets:   prometheus.ExponentialBuckets(1, 2, 12),
+			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 29),
```
Comment (lhy1024) [inline server/metrics.go:75]: how will it change metrics?

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

File: server/metrics.go (hunk 1 above)
Comment (Yisaer) [inline server/metrics.go:75]: Why '0.0005' as a bucket ?

Comment (nolouch) [reply server/metrics.go:75]: the accuracy of the bucket was too small in before, resulting in little difference in monitoring.

**Pull request closed without merging**